### PR TITLE
Fix autoholds API url that differs from the UI one

### PR DESCRIPTION
**Pull request by pablintino**
As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date

### CODE REVIEW

**Review by lewisdenny — COMMENTED** [review]
Just wondering if the file name`multinode-authold.yml` is a spelling mistake? Maybe it's meant to be `multinode-autohold.yml`

**Comment by pablintino** [comment]
recheck

File: ci/playbooks/multinode-autohold.yml [hunk 1]
```diff
@@ -3,13 +3,20 @@
   hosts: controller
   gather_facts: true
   tasks:
+    - name: Verify if "success" flag exists after successful tests execution
+      register: cifmw_success_flag
+      ansible.builtin.stat:
+        path: "{{ ansible_user_dir }}/cifmw-success"
+
     - name: Check if the current run is an authold target
```
Comment (marios) [inline ci/playbooks/multinode-autohold.yml:11]: same typo here and also line 53 below (may as well fix if you get another update here)

**Review by marios — APPROVED** [review]
seems reasonable

File: ci/playbooks/multinode-autohold.yml (hunk 1 above)
Comment (pablintino) [reply ci/playbooks/multinode-autohold.yml]: Done

**Comment by pablintino** [comment]
recheck

**Comment by pablintino** [comment]
recheck

**Comment by pablintino** [comment]
recheck

**Comment by pablintino** [comment]
recheck

**Comment by pablintino** [comment]
recheck

**Comment by pablintino** [comment]
recheck

**Comment by pablintino** [comment]
recheck

**Comment by cjeanner** [comment]
/approve
/lgtm

this will help us debugging broken jobs in a far better way :)

**Pull request merged**